sources/Context.cpp

```diff
@@ -131,8 +132,12 @@ bool Context::process() {
   }
   walkNodesMain();
 
+  algo::DynModelFilterAlgorithm dynModelFilterAlgorithm(dynamicDataBaseManager_.assembling(),
+                                                        generators_,
+                                                        dynamicModels_.models);
+  dynModelFilterAlgorithm.removeRpclInGeneratorsAndSvcIfMissingConnexionToSvc();
```

Comment:
those 2 calls should be inside the filter

Reply:
You mean I should merge removeRpclInGeneratorsAndSvcIfMissingConnexionToSvc() and filterPartiallyConnectedDynamicModels() methods in the same method ?

Reply:
No i mean that the details of what is done in the filtering should be called inside DynModelFilterAlgorithm class

Reply:
The 2 calls are DynModelFilterAlgorithm class methods